src/Numerics/Mesh/Topologies.jl

```diff
@@ -1020,38 +1026,83 @@ function cubedshellunwarp(x1, x2, x3)
     fdim = argmax(abs.((x1, x2, x3)))
 
     if fdim == 1 && x1 < 0
-        # (-R, *, *) : Face I from Ronchi, Iacono, Paolucci (1996)
+        # (-R, *, *) : formulas for Face I from Ronchi, Iacono, Paolucci (1996)
+        #              but for us face II of the developed net of the cube
         a, b, c = g(-R, x2 / x1, x3 / x1)
     elseif fdim == 2 && x2 < 0
-        # ( *,-R, *) : Face II from Ronchi, Iacono, Paolucci (1996)
+        # ( *,-R, *) : formulas for Face II from Ronchi, Iacono, Paolucci (1996)
+        #              but for us face III of the developed net of the cube
         b, a, c = g(-R, x1 / x2, x3 / x2)
     elseif fdim == 1 && x1 > 0
-        # ( R, *, *) : Face III from Ronchi, Iacono, Paolucci (1996)
+        # ( R, *, *) : formulas for Face III from Ronchi, Iacono, Paolucci (1996)
+        #              but for us face IV of the developed net of the cube
         a, b, c = g(R, x2 / x1, x3 / x1)
     elseif fdim == 2 && x2 > 0
-        # ( *, R, *) : Face IV from Ronchi, Iacono, Paolucci (1996)
+        # ( *, R, *) : formulas for Face IV from Ronchi, Iacono, Paolucci (1996)
+        #              but for us face I of the developed net of the cube
         b, a, c = g(R, x1 / x2, x3 / x2)
     elseif fdim == 3 && x3 > 0
-        # ( *, *, R) : Face V from Ronchi, Iacono, Paolucci (1996)
+        # ( *, *, R) : formulas for Face V from Ronchi, Iacono, Paolucci (1996)
+        #              and the same for us on the developed net of the cube
         c, b, a = g(R, x2 / x3, x1 / x3)
     elseif fdim == 3 && x3 < 0
-        # ( *, *,-R) : Face VI from Ronchi, Iacono, Paolucci (1996)
+        # ( *, *,-R) : formulas for Face VI from Ronchi, Iacono, Paolucci (1996)
+        #              and the same for us on the developed net of the cube
         c, b, a = g(-R, x2 / x3, x1 / x3)
     else
-        error("invalid case for cubedshellunwarp: $a, $b, $c")
+        error("invalid case for equiangular_cubed_shell_unwarp: $a, $b, $c")
     end
 
     return a, b, c
 end
 
+"""
+    equidistant_cubed_shell_warp(a, b, c, R = max(abs(a), abs(b), abs(c)))
+
+Given points `(a, b, c)` on the surface of a cube, warp the points out to a
+spherical shell of radius `R` based on the equidistant gnomonic grid outlined in
+[Rancic1996](@cite) and [Nair2005](@cite)
+"""
+function equidistant_cubed_shell_warp(a, b, c, R = max(abs(a), abs(b), abs(c)))
+
+    r = hypot(a, b, c)
+
+    x1 = R * a / r
+    x2 = R * b / r
+    x3 = R * c / r
+
+    return x1, x2, x3
+end
+
+"""
+equidistant_cubed_shell_unwarp(x1, x2, x3)
```

Comment:
```suggestion
    equidistant_cubed_shell_unwarp(x1, x2, x3[, R])
```
Can we document the arguments here?

Reply:
Thanks @charleskawczynski I believe I improved the string. Let me know if it is not clear enough. 
I'm not quite sure I understand your suggested snippet of code. Would in that case `x3` be an array?

Reply:
The `[, R]` is just to indicate an optional argument.

Reply:
The 4 spaces before `equidistant_cubed_shell_unwarp` is more important, since it's needed for Documenter.jl to recognise and format `equidistant_cubed_shell_unwarp` as code.

Reply:
Thanks Charlie!

Reply:
Added the indentation